package.json

```diff
@@ -28,49 +28,43 @@
   },
   "resolutions": {
     "@types/express": "^4.17.7",
-    "@types/serve-static": "^1.13.10",
-    "apollo-server-caching": "^0.7.0",
-    "apollo-server-core": "^3.12.0",
-    "apollo-server-env": "^3.1.0",
-    "apollo-server-plugin-base": "^3.7.2",
-    "apollo-server-types": "^3.8.0"
+    "@types/serve-static": "^1.13.10"
```

Comment:
are these still needed after updating the libs ?

Reply:
nope, removed